backend/backend.go

```diff
@@ -157,8 +158,8 @@ type Backend struct {
 	coins     map[coinpkg.Code]coinpkg.Coin
 	coinsLock locker.Locker
 
-	accounts     []accounts.Interface
-	accountsLock locker.Locker
+	accounts                []accounts.Interface
+	accountsAndKeystoreLock locker.Locker
```

Comment:
Normally, you put the mutex right above what it guards for clarity, separated by blank lines:

```go
type Backend struct {
  ...

  accountsKeystoreLock locker.Locker
  accounts []accounts.Interface
  keystore keystore.Keystore

  log
  ...
}
```

Reply:
:heavy_check_mark: